sdks/go/pkg/beam/testing/passert/passert.go

```diff
@@ -44,6 +45,21 @@ func Diff(s beam.Scope, a, b beam.PCollection) (left, both, right beam.PCollecti
 	return beam.ParDo3(s, &diffFn{Type: beam.EncodedType{T: t.Type()}}, imp, beam.SideInput{Input: a}, beam.SideInput{Input: b})
 }
```

Comment:
I want to call out that we no longer actually use this ourselves, so its theoretically a candidate for deprecation/removal. With that said, I think this is both public and usable enough that people might have dependencies on it, so I don't _really_ think that's a good idea. Mostly just had the thought and wanted to see how others feel about it.